.github/workflows/build.yaml

```diff
@@ -0,0 +1,84 @@
+name: Build
+
+on:
+  push:
+    branches: [ main, master, develop ]
```

Comment:
I can see why the (existing) `master` or even the to-be-created `main` branch would make sense, but why do we have a develop branch here?